haiku-apps/minimizeall/minimizeall-1.0.0.recipe

```diff
@@ -1,15 +1,18 @@
 SUMMARY="Minimize all your apps with the press of a button!"
-DESCRIPTION="
+DESCRIPTION="
 MinimizeAll is a small app that minimizes all your apps with the press of a \
 button on the Deskbar, or with a keypress. The app contains two parts: A \
 standalone app that minimizes the whole app, and a replicant for your Deskbar. \
 Both do the same: Minimize all your apps!"
-HOMEPAGE="https://github.com/HaikuArchives/MinimizeAll/"
+HOMEPAGE="http://github.com/HaikuArchives/MinimizeAll"
 COPYRIGHT="2001-2008 Werner Freytag
 	2009 Haiku"
 LICENSE="MIT"
-REVISION="3"
-SOURCE_URI="git://github.com/HaikuArchives/MinimizeAll.git#5afe6d4fab8f7e2a33727ef9152578b7d0c836ec"
+REVISION="4"
+COMMIT="5afe6d4fab8f7e2a33727ef9152578b7d0c836ec"
+SOURCE_URI="$HOMEPAGE/archive/$COMMIT.tar.gz"
+CHECKSUM_SHA256="b3e8d78e7e7ec723c51b63fac1151ff84a88e4091f29b1204418caf64ce83b16"
+SOURCE_DIR="minimizeall-$COMMIT"
```

Comment:
I would replace
```
REVISION="4"
COMMIT="5afe6d4fab8f7e2a33727ef9152578b7d0c836ec"
SOURCE_URI="$HOMEPAGE/archive/$COMMIT.tar.gz"
CHECKSUM_SHA256="b3e8d78e7e7ec723c51b63fac1151ff84a88e4091f29b1204418caf64ce83b16"
SOURCE_DIR="minimizeall-$COMMIT"
```
by
```
REVISION="3"
srcGitRev="5afe6d4fab8f7e2a33727ef9152578b7d0c836ec"
SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
CHECKSUM_SHA256="b3e8d78e7e7ec723c51b63fac1151ff84a88e4091f29b1204418caf64ce83b16"
SOURCE_DIR="minimizeall-$srcGitRev"
```
That's because srcGitRev is imho a slightly better choice than COMMIT, since:
* there are already 36 recipes using "srcGitRev", compared to only 4 which are using "COMMIT";
* contrary to "COMMIT", which looks like a reserved HaikuPorter variable, "srcGitRev" makes it clear that it is not a reserved but a custom variable.

And since the resulting SOURCE_URI is unchanged we don't really need to bump REVISION.

But if another contributor says that COMMIT is ok and/or that we should bump REVISION then that's OK for me too ;-)